pkgs/servers/sql/postgresql/generic.nix

```diff
@@ -320,7 +320,7 @@ let
       description = "Powerful, open source object-relational database system";
       license     = licenses.postgresql;
       changelog   = "https://www.postgresql.org/docs/release/${finalAttrs.version}/";
-      maintainers = with maintainers; [ thoughtpolice danbst globin ivan ma27 wolfgangwalther ];
+      maintainers = with maintainers; [ danbst globin ivan ] ++ teams.postgres.members;
```

Comment:
I still do use postgres-with-nix behind the scenes and even did some commercial work on it last year, but mostly it's just because I'm spread very thin and don't have time to give it singular focus and many of the PRs require much more attention than I have available.

I'm happy to remain on the maintainers list as a point-of-contact for questions though.